app/renderer/js/utils/config-util.js

```diff
@@ -50,7 +50,7 @@ class ConfigUtil {
 			this.setConfigItem(key, defaultValue);
 			return defaultValue;
 		} else {
-			return value;
+			return value.value;
```

Comment:
What's the story behind this change? Can you add a comment here or in the commit message about this.